src/Tizen.NUI/src/public/Window/WindowEvent.cs

```diff
@@ -1224,6 +1224,79 @@ private void OnAuxiliaryMessage(IntPtr kData, IntPtr vData, IntPtr optionsArray)
             }
         }
 
+        /// <summary>
+        /// AccessibilityHighlightArgs
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class AccessibilityHighlightEventArgs : EventArgs
```

Comment:
Currently, we should remove all warning messages newly created by the patch.
But.. all event arguments here have same warnings :
> Do not nest type AccessibilityHighlightEventArgs. Alternatively, change its accessibility so that it is not externally visible.

I think we'd better handle this on another PR?
@dongsug-song , @Jaehyun-Cho , @everLEEst , @taehyub  Could you give us your opinions?